domains/DataCollection/Forms/index.js

```diff
@@ -1,14 +1,45 @@
-// Make this render but switch between forms
 import React from 'react';
+import { View } from 'react-native';
+import { Text, Card } from 'react-native-paper';
 
 import IdentificationForm from './IdentificationForm';
+import SupplementaryForm from './SupplementaryForm';
 
-const Form = ({ navigation, scrollViewScroll, setScrollViewScroll }) => (
-  <IdentificationForm
-    navigation={navigation}
-    scrollViewScroll={scrollViewScroll}
-    setScrollViewScroll={setScrollViewScroll}
-  />
-);
+const Forms = (props) => {
+  const {
+    navigation, selectedForm, setSelectedForm, scrollViewScroll, setScrollViewScroll
+  } = props;
+  return (
+    <View>
+      {selectedForm === 'id' && (
```

Comment:
We're essentially controlling "navigation" by hiding and revealing components. By having this simple control in one parent component, we make it much simpler to add new forms or elements to the data collection domain.